CppXAMLSample/MarkupSample/MainPage.xaml.cpp

```diff
@@ -0,0 +1,83 @@
+﻿#include "pch.h"
+#include "MainPage.xaml.h"
+#if __has_include("MainPage.g.cpp")
+#include "MainPage.g.cpp"
+#endif
+
+
+#include <windows.h>
+#include <stdio.h>
+#include <strsafe.h>
+#include <winevt.h>
+#include <utility>
+#include <algorithm>
+#include <iostream>
+#include <chrono>
+#undef max
+
+using namespace winrt;

```

Comment:
```cpp
namespace winrt
{
    using namespace ...
}
```

Reply:
ok, but there are types with the same name in MUX and WUX, so I still need the namespace alias for WUX to disambiguate